src/InvoiceGenerator/InvoiceGenerator/InvoiceDBContext.cs

```diff
@@ -12,7 +12,7 @@ public class InvoiceDBContext : DbContext
 
         protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
-            optionsBuilder.UseSqlServer(@"");
+            optionsBuilder.UseSqlServer(@"Server=tcp:invoice-generator.database.windows.net,1433;Initial Catalog=invoice-generator;Persist Security Info=False;User ID=ef-user;Password=x3Dh0zfg2s1k;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;");
```

Comment:
this is on me for not teaching you how to abstract this away and read it from config. Hopefully we can cover this during this week.